cookbook/remote_access/remote_workflow.py

```diff
@@ -14,4 +14,29 @@
 One difference between running a task and a workflow via launchplans is that launchplans cannot be associated with a
 task. This is to avoid triggers and scheduling.
 
+FlyteRemote:
+========================
+
+Workflows and launchplans can be run through FlyteRemote programmatically.
```

Comment:
Workflows can be executed with FlyteRemote because under the hood it fetches and triggers a default launch plan.